packages/clay-data-provider/src/ClayDataProvider.js

```diff
@@ -257,15 +268,6 @@ ClayDataProvider.STATE = {
 		referrer: Config.string(),
 	}),
 
-	/**
-	 * Flag to define how often to refetch data (ms)
-	 * @instance
-	 * @default 0
-	 * @memberof ClayDataProvider
-	 * @type {?(number|undefined)}
-	 */
-	requestPolling: Config.number().value(0),
```

Comment:
hey @4lejandrito I understand that for some use cases this would not do much feeling, but we may have cases where you want the data to be updated every minute for example... with DataProvider we intend to use it in other places, such as ClayCharts. I would like to keep this API to cover these cases.